modules/phase_field/doc/content/modules/phase_field/Nucleation/DiscreteNucleation.md

```diff
@@ -27,16 +27,78 @@ adaptivity, and a postprocessor:
   that triggers refinement at the point of nucleus insertion.
 - [`DiscreteNucleationTimeStep`](postprocessors/DiscreteNucleationTimeStep.md) - a
   postprocessor to provide a time step limit for new nuclei to use with
-  [IterationAdaptiveDT](/IterationAdaptiveDT.md)
+  [IterationAdaptiveDT](/IterationAdaptiveDT.md).
+- [`DiscreteNucleationData`](postprocessors/DiscreteNucleationData.md) - a
+  postprocessor to provide diagnostic data on nucleation events.
+- [DiscreteNucleationAux](/DiscreteNucleationAux.md) - an auxkernel to map the
+  [`DiscreteNucleationMap`](/DiscreteNucleationMap.md) to an auxvariable field.
+- [DiscreteNucleationForce](/DiscreteNucleationForce.md) - a kernel to map the
+  [`DiscreteNucleationMap`](/DiscreteNucleationMap.md) to a variable field in
+  combination with a [Reaction](/Reaction.md) kernel.
 
-## Discussion
+## Nucleation approaches
 
-The nucleation free energy penalty is added to the physical free energy
-contributions of the system using a
-[`DerivativeSumMaterial`](/DerivativeSumMaterial.md)
+All approaches require the use of an inserter, and a map user object. Advanced
+users may chose to derive their own userobjects from the provided classes to,
+for example, implement maps that realize different nucleus shapes than the
+smooth interface spheres provided by
+[`DiscreteNucleationMap`](/DiscreteNucleationMap.md).
 
-## Example
+Multiple options exist for applyting the nucleus data to the variable fields in
+the simulation.
 
-An example input file is located at
+### Free energy penalty based nucleation
 
-!listing modules/phase_field/examples/nucleation/refine.i
+The free energy penalty based approaces eschew directly modifying conserved
+concentration and non-conserved order parameter fields. Instead they bias the
+thermodynamocs of the system in a way that drives the formation of nuclei at the
+locations provided by the discrete nucleation system. Simply stated the local
+free energy density is modified to make the nucleated state a lower energy
+state. This causes solute to diffuse toward the nucleation site (in the
+conserved case) brings about a change in a non-conserved order parameter field.
```

Comment:
"This causes solute to diffuse toward the nucleation site (in the conserved case) brings about a change in a non-conserved order parameter field."   This is confusing, I think you're missing some words.

Reply:
it is missing an `and`